### ISSUE TITLE
Simplified listing for 'Contributors' file

### ISSUE THREAD

**Issue by ghost** [kamilion/customizer#124]
This is a follow up from issue #121 regarding translatable 'About'. This would be the preliminary step for implementing **without translatable 'About'**.

To this date, the contributors' listing is hardcoded in Customizer GUI and looks like this:

![gui-credits-tab](https://cloud.githubusercontent.com/assets/1223304/18644148/bfddc3c6-7eda-11e6-83f3-4ccd7437bc9c.png)

The grouping of contributors under multiple categories, such as "Development team", "Git commits", "Translators", etc., introduces few problems, notably:
1. Difficult to apply translation for the categories, when running Customizer GUI in different locale i.e. other than English;
2. Difficult to locate contributors' name, regardless for simple viewing or clarifying whether the contributor's name has been added or not;
3. Possible to be listed under "wrong" category and "unfair" ordering;

Therefore, we need a **simplified listing**.

The simplified listing should include only full name and nickname (username) and arranged in alphabetical order. Then, the listing should look similar to the following screenshot.

![ctmz-gui-about-credits-2](https://cloud.githubusercontent.com/assets/1223304/18614531/9aad83fc-7dc2-11e6-9d5a-4c64816fd9c3.png)

Note that the simplified listing does _not_ include changes to the appearance of Customzier GUI as shown in the screenshot above; The changes to GUI will be applied sometime later. For time being, the simplified listing is applicable to the `Contributors` file itself only.

I hope to adapt the simplified listing in the patched release, either 4.1.3 or 4.1.4.

**While this issue will be solved in patched release, it is still part of the future release 4.2.0.**

**Comment by ghost** [comment]
I have prepared the simplified listing, which is fullest and latest to this date as below.

```
Annemieke den Teuling 'Anne'
Ayman Singh 'aymanim'
Graham Cantin 'kamilion'
Ivailo Monev 'fluxer/SmiL3y'
Michal Glowienka 'eloaders'
mic 'hreikin'
Minwook Shin 'minwook-shin'
Mubiin Kimura 'clearkimura'
Per Guth 'pguth'
Thiago Abreu 'thiagoabreu'
Username 'abrsdk'
Username 'AndrieVorster'
Username 'AwlsomeAlex'
Username 'bisrael8191'
Username 'bkkcoins' ?
Username 'bmix'
Username 'brainsucker'
Username 'brylie'
Username 'carlossoy'
Username 'confido117'
Username 'elacheche'
Username 'fsfarah'
Username 'GlassSources'
Username 'gmstyle'
Username 'Gubolin'
Username 'gururea'
Username 'hrasadi'
Username 'Israphel'
Username 'jmelton98'
Username 'kalabantas'
Username 'kerroncheddie'
Username 'kovica'
Username 'luzam'
Username 'LYF610400210'
Username 'magowiz'
Username 'mbnoimi'
Username 'mitsuzero'
Username 'optrusty'
Username 'OsdKnights'
Username 'Per'
Username 'porst17'
Username 'psycato'
Username 'ranseyer'
Username 'Rdrit'
Username 'ShvonderSiN'
Username 'ss4'
Username 'sweeneyk'
Username 'sylviette'
Username 'thomasstevenson'
Username 'tienvx'
Username 'unknown/Shiki' ?
Username 'UriHerrera'
Username 'wheatgluten'
Username 'wjwwood'
Username 'yassen-itlabs'
Username 'Yes'
Username 'zero0nee'
Username 'zxuser'
```

Note that `?` doesn't require any translation to explain. The symbol indicates either:
- username has been deleted or no longer exists on GitHub; 
- username is made annonymous by itself; or
- username cannot be verified for other than what has been recorded by commit log.

Currently, we have this many of contributors in above listing.

```
$ cat contributors-full.txt | sort -n > contributors-sort.txt
$ cat contributors-sort.txt | wc -l
58
```

This listing may be updated at any time for correction and completion, before getting this listing into `Contributors` file in the patched release 4.1.3 later. Besides this, the listing itself has been simplified and therefore I consider this issue is now 'fixed'.

**If anyone noticed someone is missing from above listing, leave a comment at below.**

**Comment by ghost** [comment]
Fixed in branch repository.
